templates/plugin.js

```diff
@@ -9,9 +10,8 @@ Middleware.auth = authMiddleware
 export default function (ctx, inject) {
   // Options
   const options = <%= JSON.stringify(options.options, null, 2) %>
-
   // Create a new Auth instance
-  const $auth = new Auth(ctx, options)
+  const $auth = new Auth(ctx, Vue.set, options)
```

Comment:
Maybe we can simply pass it to options? (and access from `options._vueSet`)  this way we don't need a new argument to class contractors.